memberscr.js

```diff
@@ -0,0 +1,52 @@
+// Code to get params from
+// https://stackoverflow.com/questions/901115/how-can-i-get-query-string-values-in-javascript/901144#901144
+
+const params = new Proxy(new URLSearchParams(window.location.search), {
+  get: (searchParams, prop) => searchParams.get(prop),
+});
+
+const id = params.name;
```

Comment:
Proxies are a very advanced feature of JavaScript that are probably a bit overkill here. It might be easier to do something like:

```js
const urlParams = URLSearchParams(window.location.search);
const id = urlParams.get("id");
```

Reply:
I don't know what this is, next time there's an ObieSource Discord hangout I'd be interested in chatting about it.